packages/toggle/components/toggle.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from "react";
+import { style } from "../style";
+
+export interface IToggleState {
+  isPrimary: boolean;
+}
+
+export interface IToggleProps {
+  primaryContent: JSX.Element | string;
+  secondaryContent: JSX.Element | string;
+}
+
+export class Toggle extends React.PureComponent<IToggleProps, IToggleState> {
+  constructor(props) {
+    super(props);
+
+    this.handleClick = this.handleClick.bind(this);
+    this.state = {
+      isPrimary: true
+    };
+  }
+
+  handleClick() {
+    this.setState({
+      isPrimary: !this.state.isPrimary
+    });
+  }
+
+  public render() {
+    const { primaryContent, secondaryContent } = this.props;
+    let content = primaryContent;
+
+    if (!this.state.isPrimary) {
+      content = secondaryContent;
+    }
+
+    return (
+      <div className={style} onClick={this.handleClick}>
```

Comment:
This isn't keyboard-navigable as-is. We should either do this as a `input[type="checkbox"]` in disguise, or add the correct ARIA roles and a `tabindex` to it, as well as keyboard handlers to toggle the state when Enter or Spacebar are pressed.

Reply:
Agree, I will add aria.